<ui version="4.0" >
 <class>ExportStateTableDlg</class>
 <widget class="QDialog" name="ExportStateTableDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>State Table Export Options</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QCheckBox" name="cb_includeout" >
           <property name="text" >
            <string>Include asyncronous output</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_resolve_invert" >
           <property name="text" >
            <string>Resolve inverted conditions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Q3ButtonGroup" name="bg_orientation" >
           <property name="title" >
            <string>Orientation</string>
           </property>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QRadioButton" name="rb_orientation0" >
              <property name="text" >
               <string>Rows: States / Columns: Events</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_orientation1" >
              <property name="text" >
               <string>Rows: Events / Columns: States</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>44</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_export" >
         <property name="text" >
          <string>Export</string>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_cancel" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cb_includeout</tabstop>
  <tabstop>rb_orientation0</tabstop>
  <tabstop>rb_orientation1</tabstop>
  <tabstop>pb_export</tabstop>
  <tabstop>pb_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cb_includeout</sender>
   <signal>clicked()</signal>
   <receiver>ExportStateTableDlg</receiver>
   <slot>includeOutputsClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_export</sender>
   <signal>clicked()</signal>
   <receiver>ExportStateTableDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ExportStateTableDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bg_orientation</sender>
   <signal>clicked(int)</signal>
   <receiver>ExportStateTableDlg</receiver>
   <slot>orientationClicked(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_resolve_invert</sender>
   <signal>clicked()</signal>
   <receiver>ExportStateTableDlg</receiver>
   <slot>resolveInvertedClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
